rust/src/lib.rs

```diff
@@ -397,7 +397,10 @@ pub mod ffi {
         fn treefile_new(filename: &str, basearch: &str, workdir: i32) -> Result<Box<Treefile>>;
         fn treefile_new_empty() -> Result<Box<Treefile>>;
         fn treefile_new_from_string(buf: &str, client: bool) -> Result<Box<Treefile>>;
-        fn treefile_new_from_fields(packages: &Vec<String>) -> Result<Box<Treefile>>;
+        fn treefile_new_from_fields(
+            packages: &Vec<String>,
+            override_remove: &Vec<String>,
```

Comment:
With just *two* arguments, this is fine.  But I suspect in the future what we really want is to use `treefile_new_empty()` and then e.g. `set_packages()` and `set_overrides_remove()` etc.

Reply:
Yeah, that's nicer. Yup, I'm working on adding more args there, so will just switch over to that.